packages/wallet/package.json

```diff
@@ -28,11 +28,11 @@
     "prettier:write": "npx prettier --write 'src/**/*.{ts,tsx}'",
     "start": "npm run clearContracts && node scripts/start.js",
     "storybook": "start-storybook -p 9001 -c .storybook",
-    "test:app": "npx jest -c ./config/jest/jest.config.js",
-    "test:ci:app": "yarn test:app --all --ci --runInBand",
-    "test:ci:contracts": "yarn contracts:compile && yarn test:contracts --all --ci --runInBand --detectOpenHandles --bail",
+    "test:app": "npx jest --runInBand -c ./config/jest/jest.config.js",
```

Comment:
@cf19drofxots Is there a reason we have to use `--runInBand` here? Slows app tests down quite a bit!

Reply:
Only reason was [this comment](https://github.com/statechannels/monorepo/pull/175#pullrequestreview-307982556)

If it's no longer applicable and the tests run fine without that flag then we should remove it.

Reply:
I think that issue only applies to the contract tests so I think we should be fine to remove it for `test:app`. (They ran fine for me locally without it). I'll update it in one of my PRs.

Reply:
FYI the issue might have been [resolved](https://github.com/ethers-io/ethers.js/issues/613#issuecomment-547895782) in version 4.0.39. @lalexgap I suggest trying to upgrade ethers-js and running all tests in parallel.